SQL/New_patches/2021-06-17_mnc2bids_Config_Settings.sql

```diff
@@ -0,0 +1,12 @@
+--MINC to BIDS converter settings
```

Comment:
```suggestion
-- MINC to BIDS converter settings
```

Not having the space make the patch impossible to source for some reason...